.github/workflows/CI.yml

```diff
@@ -332,7 +331,7 @@ jobs:
         shell: bash -l {0}
         run: |
             cd integration_tests
-            ./run_tests.py -b cpython_py c_py
+            ./run_tests.py -b cpython c_py
```

Comment:
Minor concern: We already test `cpython` in `text.xsh` (which is called in `LPython CI / LPython CI (3.10, ubuntu-latest) (pull_request)`). Do we need to test `cpython` again here?

Reply:
We don't, but I think it doesn't hurt to ensure the extra packages (sympy in this case) don't break anything. The tests run quickly.

Also I was thinking if the Python interop tests fail, but other tests pass, that we can infer what went wrong, as opposed to (all) the Python interop tests failing.

We can add it back in later, if needed.

Reply:
>I think it doesn't hurt to ensure the extra packages (sympy in this case) don't break anything.

We have `numpy` in this case, which is already/by-default available in the `lp` conda environment. For `sympy` there is a separate CI Job called `Run SymPy tests`.

```
Also I was thinking if the Python interop tests fail, but other tests pass, that we can infer what went wrong, as opposed to (all) the Python interop tests failing.

We can add it back in later, if needed.
```
Yes, we can add it back in later, if needed. Also as per the current `main` branch, it seems:
- If there is an issue in **non** `python-interop` test case, then I think three CI jobs might fail 
    - LPython CI (3.10, ubuntu-latest)
    - LPython CI (3.10, macos-latest)
    - Test CPython Interop (`@pythoncall`)
 - If there is an issue in `python-interop` tests, then I think the same three CI jobs might fail
     - LPython CI (3.10, ubuntu-latest)
    - LPython CI (3.10, macos-latest)
    - Test CPython Interop (`@pythoncall`)

Reply:
Yes, the Python interop in pure Python mode is always tested with the rest of the Python tests. While the Python interop in the C mode is tested in separate CI jobs. I think that's fine.